fdp/fdp.py

```diff
@@ -312,4 +314,4 @@ def get(self):
 
 def run_app(host, port, endpoint):
     initGraph(host=host, port=port, endpoint=endpoint)
-    app.run(host=host, port=port, debug=True)
\ No newline at end of file
+    return app
```

Comment:
Why return app and not run it here? And shouldn't we rename the function from `run_app` to something like `build_app` or `init_app`?

Reply:
Flask's `app.run` is not recommended for production running. To run the app in another framework, it's necessary to get the returned `app`. 
Indeed, renaming to `init_app` is better.

Reply:
Ok, then let's just rename it :-)

Reply:
Sure, will rename it.